general/getting-started-in-tech.md

```diff
@@ -0,0 +1,99 @@
+
+Below we got into more depth about these roles, and what languages you would need to learn.
+
+## Front-end Development
+
+As a front-end developer you would be responsible for the code that runs in the web browser and displays information to visitors. It is typically the practice of writing HTML, CSS and JavaScript.
+
+#### HTML (Hyper Text Markup Language)
+
+HTML is the language used to build websites. All text and content that you see on the internet is built using HTML.
+
+#### CSS (Cascading Style Sheets)
+
+CSS is used alongside HTML to style the page and make it look good.
+
+
+#### JavaScript
+
+JavaScript is used to change what is displayed on a web page in response to user activity.
+
+## Back-end Development
+
+As a back-end developer you would be responsible for the code that runs on the server. This type of development usually consists of three parts: a server, an application, and a database. Many back-end developers know front-end languages such as HTML and CSS but need to use languages such as Ruby on Rails, Python, PHP, Java, and .NET to get the job done.
+
+Front-end and Back-end development overlap a fair amount so starting in one role doesn’t mean that you can't switch or learn the other, should you change your mind further down the line. They also have strong ties to other areas of technology. For instance, front-end development shares a lot with mobile app development; back-end development with data science.
+
+#### Ruby
+Ruby is an object oriented scripting programming language typically used for building web applications. The most common implementation of Ruby is Rails web, the development framework built with Ruby.
+
+#### Python
+Python is the preferred option for data science thanks to scientific libraries like NumPy and pandas.
+
+#### Node.js
+Node.js is an open-source, cross-platform, back-end JavaScript runtime environment that runs on the V8 engine and executes JavaScript code outside a web browser.
+
+#### PHP
+PHP is a general-purpose scripting language especially suited to web development. It is a server side scripting language that is embedded in HTML. It is used to manage dynamic content, databases, session tracking, even build entire e-commerce sites. Also commonly used with WordPress so if custom WordPress themes interests you, then learn PHP.
+
+#### SQL
+SQL is a domain-specific language designed for managing data held in a relational database management system.
+
+## Full-stack Development
+
+A full-stack web developer is a person who can develop both client (front-end) and server (back-end) software. For this we suggest learning any of the languages under front-end development and back-end development. Below are some further subjects that relate to both:
+
+#### Testing
+Why do you want to test and what different types of tests you can do:
+- Unit testing
+- Integration testing
+- End to end testing
+- Acceptance testing: Cucumber/Gherkin + shared understanding with business
```

Comment:
@KimberleyCook if we are adding libraries + examples here shouldn't we add it to the other entries as well?

Reply:
So just say `acceptance testing` ??

Reply:
@KimberleyCook yeah. I think we either add examples or a small paragraph explaining what they are to all of them or none of them, it feels weird that this kind of testing is singled out. For a newbie probably none of the other terms will make sense either.